admin/acceptance.py

```diff
@@ -91,6 +91,7 @@ def check_result(f):
             FLOCKER_ACCEPTANCE_CONTROL_NODE=control_node.address,
             FLOCKER_ACCEPTANCE_AGENT_NODES=':'.join(
                 node.address for node in agent_nodes),
+            FLOCKER_ACCEPTANCE_VOLUME_BACKEND='loopback',
```

Comment:
This will get rid of it in the short term, but hopefully we'll get it back by the end of the sprint.